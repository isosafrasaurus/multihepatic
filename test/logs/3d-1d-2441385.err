Running "module reset". Resetting modules to system default. The following $MODULEPATH directories have been removed: None
set -u

# Find the runtime
APPTAINER_BIN="$(command -v apptainer || true)"
[[ -z "$APPTAINER_BIN" ]] && APPTAINER_BIN="$(command -v singularity || true)"
if [[ -z "$APPTAINER_BIN" ]]; then
  echo "[JOB][ERROR] apptainer/singularity not found on PATH after module load." >&2
  exit 127
fi

# Scrub host preloads so they can't contaminate the container
unset LD_PRELOAD LD_AUDIT || true
export APPTAINERENV_LD_PRELOAD=""
export APPTAINERENV_LD_AUDIT=""

# Job inputs from the wrapper (sed replaces these placeholders)
IMAGE_URI="docker://ghcr.io/isosafrasaurus/tacc-mvapich2.3-python3.12-graphnics:latest"
REPO_URL="https://github.com/isosafrasaurus/3d-1d"
RUN_SCRIPT="test.py"
TASKS=1

# Use repo in current directory if present; otherwise clone into WORK/SCRATCH for this job
if [[ -f "./${RUN_SCRIPT}" ]]; then
  REPO_DIR="$PWD"
else
  REPO_DIR="${WORK:-${SCRATCH:-$PWD}}/3d-1d-${SLURM_JOB_ID}"
  mkdir -p "$REPO_DIR"
  module load git >/dev/null 2>&1 || true
  if ! command -v git >/dev/null 2>&1; then
    echo "[JOB][ERROR] git is required to clone ${REPO_URL} but was not found." >&2
    exit 2
  fi
  echo "[JOB] Cloning ${REPO_URL} into ${REPO_DIR} ..."
  git clone --depth 1 "$REPO_URL" "$REPO_DIR"
fi
Cloning into '/work2/10756/piercezhang9871/stampede3/3d-1d-2441385'...
cd "$REPO_DIR"
echo "[JOB] Repo dir: $PWD"

# Avoid occasional Apptainer runtime dir quirks on shared systems
unset XDG_RUNTIME_DIR || true
export APPTAINER_CACHEDIR="${SCRATCH:-$HOME}/.apptainer/cache"
mkdir -p "$APPTAINER_CACHEDIR"

# Trace only the interesting part (container run), not module plumbing
set -x
"$APPTAINER_BIN" --version
+ /opt/apps/tacc-apptainer/1.4.1/bin/apptainer --version

# Launch: clean env inside container; sanitize export from Slurm
srun -n "${TASKS}" --export=ALL,LD_PRELOAD=,LD_AUDIT= \
  "$APPTAINER_BIN" exec --cleanenv -B "$PWD:/workspace" "$IMAGE_URI" \
  python3 "/workspace/$RUN_SCRIPT"
+ srun -n 1 --export=ALL,LD_PRELOAD=,LD_AUDIT= /opt/apps/tacc-apptainer/1.4.1/bin/apptainer exec --cleanenv -B /work2/10756/piercezhang9871/stampede3/3d-1d-2441385:/workspace docker://ghcr.io/isosafrasaurus/tacc-mvapich2.3-python3.12-graphnics:latest python3 /workspace/test.py
INFO:    Using cached SIF image
INFO:    gocryptfs not found, will not be able to use gocryptfs
/usr/local/lib/python3.12/site-packages/ufl_legacy/__init__.py:243: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  import pkg_resources
/usr/local/lib/python3.12/site-packages/block/__init__.py:15: UserWarning: The cbc.block repository has moved to https://github.com/blocknics/cbc.block
  warnings.warn('The cbc.block repository has moved to https://github.com/blocknics/cbc.block', UserWarning)
Averaging over 224 cells:   0%|          | 0/224 [00:00<?, ?it/s]Averaging over 224 cells:  91%|█████████ | 204/224 [00:00<00:00, 2039.54it/s]Averaging over 224 cells: 100%|██████████| 224/224 [00:00<00:00, 2004.58it/s]
rc=$?
+ rc=0
set +x
+ set +x

echo "[JOB] Container exit code: $rc"
echo "[JOB] =====  END  $(date) ====="
exit "$rc"
