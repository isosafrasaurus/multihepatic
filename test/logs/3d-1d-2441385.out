[JOB] ===== BEGIN Mon Oct  6 01:32:15 PM CDT 2025 =====
[JOB] Host: c455-023.stampede3.tacc.utexas.edu
[JOB] CWD : /work2/10756/piercezhang9871/stampede3/3d-1d/test
[JOB] Cloning https://github.com/isosafrasaurus/3d-1d into /work2/10756/piercezhang9871/stampede3/3d-1d-2441385 ...
[JOB] Repo dir: /work2/10756/piercezhang9871/stampede3/3d-1d-2441385
apptainer version 1.4.1-1.el9
Missing HsMG for fract norm computing
WARNING: user expression has not supplied value_shape method or an element. Assuming scalar element.
WARNING: user expression has not supplied value_shape method or an element. Assuming scalar element.
net_flow_all = 4.721770307206342e-07
[JOB] Container exit code: 0
[JOB] =====  END  Mon Oct  6 01:32:27 PM CDT 2025 =====
